.eslintrc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  extends: ['airbnb-base', 'plugin:vue/base'],
+  extends: ['airbnb-base'],
```

Comment:
Plugin `plugin:vue/base` (`eslint-plugin-vue`) looks to be no used anymore, did you uninstall with (--save) it?

Reply:
I've removed the `eslint-plugin-vue` within the two last commits (7b669a2da9ac74ae51ea8ac9b1752afa09914a3d, 7d35e9e39efdcc3727939cc781e0c4b620f5cc76). Moreover I removed the `docs` linting process from the main `eslint` HoT script. It's already checked within separated GH Workflow.